packages/@react-aria/radio/src/useRadioGroup.ts

```diff
@@ -127,7 +126,7 @@ export function useRadioGroup(props: AriaRadioGroupProps, state: RadioGroupState
       // https://www.w3.org/TR/wai-aria-1.2/#radiogroup
       role: 'radiogroup',
       onKeyDown,
-      'aria-invalid': validationState === 'invalid' || undefined,
+      'aria-invalid': state.isInvalid || undefined,
```

Comment:
to stay backwards compatible, this hook should still include validationState no?